lib/uffizzi/cli/preview.rb

```diff
@@ -0,0 +1,268 @@
+# frozen_string_literal: true
+
+require 'uffizzi'
+require 'tty-spinner'
+require 'uffizzi/auth_helper'
+
+module Uffizzi
+  class CLI::Preview < Thor
+    include ApiClient
+
+    @spinner
+
+    class << self
+      def help(_shell, _subcommand)
+        Cli::Common.show_manual(:preview)
+      end
+    end
+
+    desc 'list', 'list'
+    def list
+      return Cli::Common.show_manual(:list) if options[:help]
+
+      run(options, 'list', nil, nil)
+    end
+
+    desc 'create', 'create'
+    def create(file_path = nil)
+      return Cli::Common.show_manual(:create) if options[:help]
+
+      run(options, 'create', file_path, nil)
+    end
+
+    desc 'delete', 'delete'
+    def delete(deployment)
+      return Cli::Common.show_manual(:delete) if options[:help]
+
+      run(options, 'delete', nil, deployment)
```

Comment:
You are passing an argument (file_path as nil) that is not supposed to be in this method. There must be a better way, using ARGV or something.